app/components/explore/side-bar.js

```diff
@@ -13,8 +13,8 @@ export default Component.extend({
   customEndDate : null,
   showFilters   : false,
 
-  hideClearFilters: computed('category', 'sub_category', 'event_type', 'startDate', 'endDate', 'location', 'ticket_type', function() {
-    return !(this.category || this.sub_category || this.event_type || this.startDate || this.endDate || this.location || this.ticket_type !== null);
+  hideClearFilters: computed('category', 'sub_category', 'event_type', 'startDate', 'endDate', 'location', 'ticket_type', 'cfs', function() {
+    return !(this.category || this.sub_category || this.event_type || this.startDate || this.endDate || this.location || this.ticket_type || this.cfs !== null);
```

Comment:
if you are passing a string to query param cfs, you can just use `|| this.cfs` here?

Reply:
the other data type are also string, hence i added it in this way.

Reply:
this.cfs !== null is true if cfs is not null, false if it is null.
so you can just write this.cfs, no need for the explicit comparison to null.